halconfig/echo-scheduler.yml

```diff
@@ -0,0 +1,33 @@
+# halconfig
+
+server:
+  port: ${services.echo-scheduler.port:8089}
+  address: ${services.echo-scheduler.host:localhost}
+
+spinnaker:
+  baseUrl: ${services.deck.baseUrl}
+  cassandra:
+     enabled: false
+  inMemory:
+     enabled: true
+
+front50:
+  baseUrl: ${services.front50.baseUrl:http://localhost:8080}
+
+orca:
+  baseUrl: ${services.orca.baseUrl:http://localhost:8083}
+
+endpoints.health.sensitive: false
+
+scheduler:
+  enabled: true
+  threadPoolSize: 20
+  triggeringEnabled: true
+  pipelineConfigsPoller:
+    enabled: true
+    pollingIntervalMs: 30000
+  cron:
+    timezone: ${global.spinnaker.timezone:America/Los_Angeles}
+
+redis:
```

Comment:
@duftler just made a change (or is going to) that adds an enabled flag

```yaml
redis:
  enabled: ${services.redis.enabled:false}
```

Could you add that here too?